MultimodalQnA/ui/gradio/multimodalqna_ui_gradio.py

```diff
@@ -236,17 +233,17 @@ def ingest_video_gen_caption(filepath, request: gr.Request):
     files = {
         "files": open(dest, "rb"),
     }
-    response = requests.post(dataprep_gen_captiono_addr, headers=headers, files=files)
+    response = requests.post(dataprep_gen_caption_addr, headers=headers, files=files)
```

Comment:
Was this a typo? So it didn't work before?

Reply:
It worked because it was spelled this way at declaration too (bottom of file), but it looked inconsistent so I changed them.